### DISCUSSION TITLE
Reply to email sent to alias

### DISCUSSION THREAD

**Discussion by Klanky42** [simple-login/app#2006] [Q&A]
Hi, 
maybe i'm stupid but I can't really understand how it's meant to work. When I send an email using alias how can I reply to that mail using Proton Mail (unlimited), when I click reply only my mailadresses I have in Proton show up, how can I be certain that they don't see my original mailadress. 

OR is meant that i should go to the Alias Contact Manager in SimpleLogin and copy that alias and compose a new mail and make it look that I replied to the mail sent to me?

I would prefer if I just could reply to mails as usual without doing the above, is that possible?
Thanks in advance, I'm just 5 months old, coming from goggle...
Yours Klanky

**Comment by silencebow** [comment]
When you reply to an email sent to an alias, you'll be sending the reply back to a SimpleLogin reverse-alias instead of directly back to the original sender. If you look at the address you're responding to, instead of something like 'noreply@discord.com' it will be something like 'noreply_at_discord_com_example@simplelogin.co'. Then SimpleLogin sends the email from your alias to the original sender, as seen below:

![image](https://github.com/simple-login/app/assets/29562312/4a5d5504-0194-493b-80f5-fdb117013ff9)

Reverse-aliases are automatically made for all addresses that send an email to the alias. You only need to make one manually on SimpleLogin if you're sending an email to a new address.

**Reply by Klanky42** [reply]
Thank you, silencebow!
At last I get it, it works like I wanted all along, thanks for your helpful answer.
Good luck with everything!
Cheers Klanky
